### PR TITLE
[TIQR-63] Misc. updates

### PR DESCRIPTION
**Pull request by okaymak**

### CODE REVIEW

**Comment by dzolnai** [comment]
You set targetSdk to 27 but I don't see any NotificationChannels being created.

File: build.gradle [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
     repositories {
+        google()
         jcenter()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.3'
-        classpath 'com.google.gms:google-services:3.0.0'
+        classpath 'com.android.tools.build:gradle:3.0.1'
+        classpath 'com.google.gms:google-services:3.1.0'
```
Comment (dzolnai) [inline build.gradle]: 3.1.1 is latest

File: glass/build.gradle [hunk 2]
```diff
@@ -20,10 +20,10 @@ android {
...
-    compile project(':shared')
-    compile 'com.embarkmobile:zxing-android-minimal:2.0.0@aar'
-    compile 'me.dm7.barcodescanner:zbar:1.5'
-    compile 'com.github.bumptech.glide:glide:4.0.0'
-    annotationProcessor 'com.github.bumptech.glide:compiler:4.0.0'
+    implementation fileTree(include: ['*.jar'], dir: 'libs')
+    implementation project(':shared')
+    implementation 'com.embarkmobile:zxing-android-minimal:2.0.0@aar'
+    implementation 'me.dm7.barcodescanner:zbar:1.5'
+    implementation 'com.github.bumptech.glide:glide:4.3.1'
```
Comment (dzolnai) [inline glass/build.gradle]: 4.5.0 is latest

File: glass/build.gradle [hunk 3]
```diff
@@ -20,10 +20,10 @@ android {
...
+    annotationProcessor 'com.github.bumptech.glide:compiler:4.3.1'
```
Comment (dzolnai) [inline glass/build.gradle]: You don't need the compiler if you are not using a generated module, which I don't see, so you can remove this.

File: android/src/main/res/drawable-v26/ic_launcher_foreground.xml [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (dzolnai) [inline android/src/main/res/drawable-v26/ic_launcher_foreground.xml]: I don't see this file used anywhere, can it be removed?

Comment (rubenweerts) [reply android/src/main/res/drawable-v26/ic_launcher_foreground.xml]: Looks like `mipmap-anydpi-v26/ic_launcher.xml` references a file with this name. But that is in the mipmap folder.

Comment (dzolnai) [reply android/src/main/res/drawable-v26/ic_launcher_foreground.xml]: Ah I see, then this comment can be ignored. Thanks for finding it.

File: glass/build.gradle (hunk 2 above)
Comment (okaymak) [reply glass/build.gradle]: Keeping it a 4.3.1 because build issues

**Comment by okaymak** [comment]
Updated

**Pull request merged**